doc/educational_decoder/harness.c

```diff
@@ -106,9 +106,15 @@ int main(int argc, char **argv) {
         return 1;
     }
 
+    dictionary_t* parsed_dict = create_dictionary();
```

Comment:
minor : `dictionary_t* const` since `parsed_dict` is not going to be changed later.